components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/db/SitePermissionsEntity.kt

```diff
@@ -37,6 +37,12 @@ internal data class SitePermissionsEntity(
     @ColumnInfo(name = "local_storage")
     var localStorage: SitePermissions.Status,
 
+    @ColumnInfo(name = "autoplay_audible")
+    var autoplayAudible: SitePermissions.Status,
+
+    @ColumnInfo(name = "autoplay_inaudible")
+    var autoplayInaudible: SitePermissions.Status,
```

Comment:
IIUC, these should default to 0 (`NO_DECISION`), so we _shouldn't_ need to migrate anything?

Reply:
Agree!